eos/nonlocal-form-factors/charm-loops-analytic_TEST.cc

```diff
@@ -221,8 +221,8 @@ class TwoLoopTest :
                     TEST_CHECK_NEARLY_EQUAL(f29c_ref[i].real(), f29c_res.real(), 150*eps);
                     TEST_CHECK_NEARLY_EQUAL(f29c_ref[i].imag(), f29c_res.imag(), 150*eps);
 
-                    TEST_CHECK_NEARLY_EQUAL(f27d_ref[i].real(), f27d_res.real(), 250*eps);
-                    TEST_CHECK_NEARLY_EQUAL(f27d_ref[i].imag(), f27d_res.imag(), 200*eps);
+                    TEST_CHECK_NEARLY_EQUAL(f27d_ref[i].real(), f27d_res.real(), 2000*eps);
+                    TEST_CHECK_NEARLY_EQUAL(f27d_ref[i].imag(), f27d_res.imag(), 2000*eps);
```

Comment:
I am 2000% happy with this, since we already enlarged the uncertainties. I think I will have to investigate a bit. Does @smeiser know on which architecture the test failed?